controllers/helpers/metrics.go

```diff
@@ -0,0 +1,88 @@
+/*
+© 2021 Red Hat, Inc. and others
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/operator-framework/operator-sdk/pkg/metrics"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/rest"
+	controllerClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func SetupMetrics(namespace string, owner metav1.Object, labels map[string]string, port int32,
```

Comment:
... then this becomes:

```suggestion
func Setup(namespace string, owner metav1.Object, labels map[string]string, port int32,
```
so users reference `metrics.Setup`.